File: ABC/ABC089/B/main.cpp
#include <bits/stdc++.h>
using namespace std;

int main() {

  int N;
  cin >> N;

  set<string> S;

  for (int i = 0; i < N; i++) {
    string S_;
    cin >> S_;
    S.insert(S_);
  }

  if (S.size() == 3) {
    cout << "Three" << endl;
  } else {
    cout << "Four" << endl;
  }

}

File: ABC/ABC089/B/main.java
import java.util.*;

public class Main {

  public static void main (String[] args) {

    Scanner sc = new Scanner(System.in);
    
    int N = sc.nextInt();
    
    String[] S = new String[N];

    for (int i = 0; i < N; i++) {
      S[i] = sc.next();
    }

    sc.close();

    String result = "Three";

    for (int i = 0; i < N; i++) {
      if (S[i].equals("Y")) {
        result = "Four";
      }
    }
    
    System.out.println(result);
 
  }

}